tests/api_tests/util.py

```diff
@@ -72,8 +80,8 @@ def assert_success(response, data=None, ignore_order=True):
 
     if data is not None:
 
-        print 'OUTPUT_DATA = %r' % (output,)
-        print 'EXPECTED_DATA = %r' % (data,)
+        print('OUTPUT_DATA = %r' % (output,))
```

Comment:
Heads up that this isn't actually calling `print` as a function. It's just passing a string wrapped in `()` to the `print` statement. It looks the same but the behavior breaks down if you're doing anything more complex than printing a single string.

It would be more correct to do `from __future__ import print_function` at the top of the module.

Reply:
Good call out! I'll fix this and make sure we have that import everywhere else we use `print`